frontend/packages/client/src/hooks/useToast.js

```diff
@@ -1,15 +1,22 @@
+import { useHistory } from 'react-router-dom';
 import Toast from 'components/common/Toast';
 import { useToast } from '@chakra-ui/react';
 
 export default function useToastHook() {
   const toast = useToast();
+  const history = useHistory();
```

Comment:
Here as well, if we have the toast just be a wrapper of whatever content, we have this hook assume the sole responsibility of managing the show and hide of toast component